tests/IBTK/CMakeLists.txt

```diff
@@ -0,0 +1,168 @@
+# 2-D Tests
+ibamr_add_tests(equal_eps
+  LIBRARIES IBAMR2d
+)
+ibamr_add_tests(hierarchy_callbacks
+  LIBRARIES IBAMR2d
+)
+ibamr_add_tests(ibtk_init
+  LIBRARIES IBAMR2d
+)
+ibamr_add_tests(ibtk_mpi
+  LIBRARIES IBAMR2d
+  CASES
+    "mpirun=8"
+)
+ibamr_add_tests(ldata_01
+  LIBRARIES IBAMR2d
+  CASES
+    "<default>"
+    "mpirun=4"
+)
+ibamr_add_tests(mpi_type_wrappers
+  LIBRARIES IBAMR2d
+)
+
+ibamr_add_2d_tests(child_integrators)
```

Comment:
This test has now been fixed on the main branch. If you rebase, this test is now dimension independent.